.github/workflows/presubmit.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Presubmit
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  build_runtime:
+    uses: ./.github/workflows/build_runtime_cmake.yml
+    with:
+      build-dir: build-runtime
+
+  test_runtime:
+    needs: build_runtime
+    uses: ./.github/workflows/test_runtime_cmake.yml
```

Comment:
I might have some misunderstandings, do we need to use environment to make sure presubmit only run for reviewed PRs before we have the ephemeral runners?